common/views/components/Images/README.md

```diff
@@ -0,0 +1,10 @@
+## Captioned image heights
```

Comment:
Reckon it might be worth sticking a purpose in here for consistency, and to make sure we're being true to using cardigan as discussed (might be worth creating a purpose for cardigan).



Something like:
```suggestion
# Purpose
To display images in an accessibly and performant way, with a contextual caption to help tell the story of why the image is placed where it is. Also contains the attribution of the image.
```